aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -268,66 +268,43 @@ public synchronized void initialize(
         threadPool.submit(() -> {
             try {
                 final String modelName = getModelName(item);
-                final ModelSchema modelSchema =
-                    modelSchemaRegistry.getModelSchemaForModelClass(modelName);
-                final SQLiteTable sqliteTable = SQLiteTable.fromSchema(modelSchema);
-                final String primaryKeyName = sqliteTable.getPrimaryKeyColumnName();
-                final SqlCommand sqlCommand;
-                final ModelConflictStrategy modelConflictStrategy;
```

Comment:
Can you delete `ModelConflictStrategy.java` now?  I like that you were able to eliminate its usage with this refactor!

Reply:
ah, i missed that. Yup we can remove it now.